verification/curator-service/ui/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
do we need to commit this file?

FWM if this was generated by a command and we'll clean up example stuff later

Reply:
Yes, twas "generated by a command and we'll clean up example stuff later"